services/UserService.php

```diff
@@ -53,6 +53,15 @@ public function getActiveUsers() {
     }
 
     /**
+     * @return array All users (active + inactive)
+     */
+    public function getAllUsers()
+    {
+        //return $this->repository->getAllUsers();
+        return ["HELLO"];
```

Comment:
Ha, I'm just such a friendly person I feel the need to say hello throughout my code